### PR TITLE
Update edge join API path and response

### PR DESCRIPTION
**Pull request by Ma-Dan**

### CODE REVIEW

File: pkg/edgeservice/handler.go [hunk 1]
```diff
@@ -136,44 +162,67 @@ func EdgeNodeJoin(request *restful.Request, response *restful.Response) {
 
 	_, ok := nodeNames[nodeName]
 	if ok {
-		log.Printf("Node name %s already used\n", nodeName)
-		response.AddHeader("Content-Type", "text/plain")
-		errMsg := fmt.Sprintf("Node name %s already used\n", nodeName)
-		response.WriteErrorString(http.StatusBadRequest, errMsg)
+		log.Printf("EdgeNodeJoin: Node name %s is already used\n", nodeName)
```
Comment (benjaminhuo) [inline pkg/edgeservice/handler.go:165]: Node name %s is already used =>
Node name %s is already in use

File: pkg/edgeservice/handler.go [hunk 2]
```diff
@@ -136,44 +162,67 @@ func EdgeNodeJoin(request *restful.Request, response *restful.Response) {
...
+		response.AddHeader("Content-Type", "text/json")
+		response.WriteHeader(http.StatusBadRequest)
+		response.WriteAsJson(&EdgeJoinResponse{
+			Code:    http.StatusBadRequest,
+			Status:  StatusFailure,
+			Message: fmt.Sprintf("Node name %s is already used\n", nodeName),
```
Comment (benjaminhuo) [inline pkg/edgeservice/handler.go:171]: Node name %s is already used =>
Node name %s is already in use

File: pkg/edgeservice/handler.go [hunk 3]
```diff
@@ -136,44 +162,67 @@ func EdgeNodeJoin(request *restful.Request, response *restful.Response) {
...
 		return
 	}
 
 	_, ok = nodeIPs[nodeIP]
 	if ok {
-		log.Printf("Node IP %s already used\n", nodeIP)
-		response.AddHeader("Content-Type", "text/plain")
-		errMsg := fmt.Sprintf("Node IP %s already used\n", nodeIP)
-		response.WriteErrorString(http.StatusBadRequest, errMsg)
+		log.Printf("EdgeNodeJoin: Node IP %s is already used\n", nodeIP)
```
Comment (benjaminhuo) [inline pkg/edgeservice/handler.go:178]: Node IP %s is already used =>
Node IP %s is already in use

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: pkg/edgeservice/handler.go (hunk 1 above)
Comment (Ma-Dan) [reply pkg/edgeservice/handler.go]: Fixed

File: pkg/edgeservice/handler.go (hunk 2 above)
Comment (Ma-Dan) [reply pkg/edgeservice/handler.go]: Fixed

File: pkg/edgeservice/handler.go (hunk 3 above)
Comment (Ma-Dan) [reply pkg/edgeservice/handler.go]: Fixed

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**